snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -603,25 +617,36 @@ func (b *Bootstrapper) process(
 		height := blk.Height()
 		b.tipHeight = max(b.tipHeight, height)
 
-		if numPreviouslyFetched/statusUpdateFrequency != numFetched/statusUpdateFrequency {
+		// Check if it's time to log progress (both progress-based and time-based frequency)
+		now := time.Now()
+		shouldLog := numPreviouslyFetched/statusUpdateFrequency != numFetched/statusUpdateFrequency &&
```

Comment:
Should we just remove the `statusUpdateFrequency` from here? It was definitely added just because we were lazy and didn't want to add another tracker (which we are doing now with `lastProgressUpdateTime`)